### ISSUE TITLE
Initial installation fails

### ISSUE THREAD

**Issue by mcgwiz** [zigg/brewdo#10]
Starting from an OSX machine without homebrew installed, I cloned the brewdo repo and, with my non-privileged account, ran `sudo make install`. This placed the brewdo scripts into /usr/local. (I then had to `chmod -R a+rX /usr/local`.)

Next, I ran `sudo brewdo install`. Unfortunately, this yielded:

```
WARNING: Improper use of the sudo command could lead to data loss
or the deletion of important system files. Please double-check your
typing when using sudo. Type "man sudo" for more information.

To proceed, enter your password, or type Ctrl-C to abort.

Password:
fatal: destination path '/usr/local' already exists and is not an empty directory.
/usr/local/bin/brewdo:280:in `cmd_clone': Homebrew clone failed (GitCloneError)
from /usr/local/bin/brewdo:291:in `cmd_install'
from /usr/local/bin/brewdo:389:in `<main>'
```

I'd like the brewdo scripts to be available to all users, but given homebrew's desire to own /usr/local, I'm not sure how to achieve this in the most maintainable way. Any tips? Considering putting brewdo in /opt/local and adding some bash init scripts into "/System/Library/User Template/English.lproj"...

**Comment by zigg** [comment]
Reviewing my README, it seems I got the installation steps backwards.  Sorry about that!

`git` is definitely unhappy about cloning into a directory that already exists, let alone one that it doesn't actually have rights to clone into (e.g. as the sandbox user cloning into a root-owned directory.)  If you don't have an existing `/usr/local`, the best way to install brewdo is actually to run `sudo ./brewdo install` from your checkout *first*.

As for the rest, since #7 was fixed, it's also now possible to run `./brewdo do make install` instead of `sudo make install` to install brewdo itself, which leaves everything nicely owned by the sandbox user.

I'll fix this issue as a documentation bug for now, with an eye toward some other enhancements to ease this path—for example, seeing if the Homebrew installer routine at https://github.com/Homebrew/install/blob/master/install#L191-L206 might allow `brewdo install` to clone Homebrew into an existing `/usr/local` (given the proper permissions, of course.)

Thanks for the report!
